boot/bootdata/hivecls.inf

```diff
@@ -24,7 +32,7 @@ HKCR,".clp","Content Type",0x00000000,"application/x-msclip"
 HKCR,"clpfile","",0x00000000,"Clipboard Element"
 HKCR,"clpfile","FriendlyTypeName",0x00020000,"@%SystemRoot%\system32\clipbrd.exe,-121"
 HKCR,"clpfile\DefaultIcon","",0x00020000,"%SystemRoot%\system32\clipbrd.exe,1"
-HKCR,"clpfile\shell\open\command","",0x00020000,"%SystemRoot%\system32\clipbrd.exe %1"
+HKCR,"clpfile\shell\open\command","",0x00020000,"%SystemRoot%\system32\clipbrd.exe ""%1"""
```

Comment:
Not sure about this one; have you tested something like:
```
clipbrd.exe "C:\Documents and settings\Administrator\Desktop\test.clp"
```
?

Reply:
```c
int main(void)
{
printf("%s\n", GetCommandLineA());
return 0;
}
```

Reply:
![clip-ignore](https://user-images.githubusercontent.com/2107452/41833012-96146de2-7889-11e8-852b-d1d568e90948.png)
Oh, no! Win2k3 ClipBook Viewer is simply ignoring the parameters. What should we do?

Reply:
On Win2k3, you can save (using the clipbrd) the clipboard contents to a file (with extension .clp).
Then try opening that file when it's inside a long-named directory with spaces.
Try also to do the same, but using ReactOS' clipbrd.exe.

Reply:
![clipbrd1](https://user-images.githubusercontent.com/2107452/41881582-2eddfb2a-791f-11e8-85d7-ef04587549d0.png)![clipbrd2](https://user-images.githubusercontent.com/2107452/41881581-2e91b8b4-791f-11e8-8874-87f3ab84a37a.png)
Both cases are failed in both of text and bitmap formats.

Reply:
I think the clipbrd program should accept both of quoted and unquoted file paths.